### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kbrandl**
Add more guidance about the NPM package for Office.js.

### CODE REVIEW

**Comment by kbrandl** [comment]
@Zlatkovsky this PR updates the README with more guidance about the NPM package for Office.js, per our recent discussion.

**Comment by ElizabethSamuel-MSFT** [comment]
@kbrandl Could you also update dev.office.com links? Thanks.

File: README.md [hunk 1]
```diff
@@ -4,20 +4,55 @@
...
+
+### Target scenarios
+
+The NPM package for Office.js is intended as a way for you to obtain your own (non-CDN) copy of the Office.js files, which you can then statically serve from your own site instead of using the CDN. This NPM package is primarily provided to address the following scenarios:
+
+1. If you are developing an add-in behind a firewall, where accessing the Office.js CDN is not possible.
+
+2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
+
+Additionally, using the alpha and beta versions of the NPM package can facilitate faster-cadence beta-testing (relative to the slower-cadence beta CDN at **<https://appsforoffice.microsoft.com/lib/beta/hosted/office.js>**).
```
Comment (Zlatkovsky) [inline README.md]: There is no "alpha".  And actually, the beta may or may not be faster-cadence (recently it's been the opposite).  I would either remove this sentence, or reframe as more of a "Additionally, if you're working with someone at Microsoft who recommends you use a specific private "adhoc" build of Office.js for testing, the NPM packages that get published off of this repo will help you easily test that code".

File: README.md [hunk 2]
```diff
@@ -4,20 +4,55 @@
...
     npm install @microsoft/office-js --save
 
-Once installed, the Office.js script reference can be used as
+<br />
+
+## Using the NPM package
+
+Installing the NPM package locally creates a set of static Office.js files in the `node_modules\@microsoft\office-js\dist` folder of the directory where you ran the `npm install` command. To use the NPM package, do the following:
+
+1. Copy the contents of this folder as-is to the destination of your choosing.
```
Comment (Zlatkovsky) [inline README.md]: I would change it to something like "Either manually or as part of a build script (e.g., `CopyWebpackPlugin` if you're using Webpack), have the files get served off of a destination of your choosing (e.g., `/assets/office-js/`)

File: README.md [hunk 3]
```diff
@@ -4,20 +4,55 @@
...
+
+2. Reference that location in a `<script>` tag within the HTML file in your add-in project.
 
-    <script src="node_modules\@microsoft\office-js\dist\office.js"></script>
+For example, if you add the contents of the `dist` folder to the `assets` directory of your web server (`https://mysite.com`), then you'd add the following `<script>` tag to your HTML file:
 
+    <script src="https://mysite.com/assets/office.js"></script>
```
Comment (Zlatkovsky) [inline README.md]: It's better to use a relative path that's relative off of the root of the url.  That way it works on localhost too
E.g.: `/assets/office-js/office.js`

**Review by Zlatkovsky — APPROVED** [review]

**Comment by Zlatkovsky** [comment]
@kbrandl , looks good on the whole.  I left a few comments, please take a look.  Once you're done, feel free to merge it in yourself.

**Comment by Zlatkovsky** [comment]
@kbrandl , one other thing:  we should clarify about the d.ts.

Namely:
1. If you are using this package for the firewall scenario and want a d.ts that precisely matches the JS contents, use the d.ts under `/dist/office.d.ts`.  Best way to to do it is via a triple-slash reference (https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).  I'm pretty sure that once you include it in just one file, it will work for all the others as well.  (Pro tip:  what I've done is just create a "references.ts" file at the root of the project and point the reference to office.d.ts there).
2. If you are using this package for **beta** (which we don't have a standalone d.ts for for now, though Alex was considering just adding an "office-js-beta" to definitelytyped), use this same as in point 1, but just make sure to update often.
3. If neither applies, just use the types from `@types/office-js` (which is identical to what's on DefinitelyTyped), and reference the CDN -- in which case you don't need this package!

For most people, they should actually fall into number 3.

**Comment by Zlatkovsky** [comment]
Kim, one last thing:  when you merge this in, could you also close https://github.com/OfficeDev/office-js/issues/62?

**Comment by kbrandl** [comment]
@Zlatkovsky thanks for the review! I've updated the README per your feedback, including adding the additional info about the d.ts. I'll merge this PR now and then will close #62.

**Pull request merged**